streamlit_drawable_canvas/frontend/src/DrawableCanvas.tsx

```diff
@@ -143,6 +143,10 @@ const DrawableCanvas = ({ args }: ComponentProps) => {
       saveState(canvas.toJSON())
     })
 
+    canvas.on("mouse:dblclick", () => {
```

Comment:
since you're adding a `mouse:dblclick` event, could you unsubscribe from it in the return callback?

Reply:
Sorry, in my original fix I did unsubscribe, but I forgot to include it here.